modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/exportimport/data/handler/CalendarPortletDataHandler.java

```diff
@@ -28,15 +35,70 @@
 	property = "javax.portlet.name=" + CalendarPortletKeys.CALENDAR,
 	service = PortletDataHandler.class
 )
-public class CalendarPortletDataHandler
-	extends CalendarAdminPortletDataHandler {
+public class CalendarPortletDataHandler extends BasePortletDataHandler {
+
+	public static final String SCHEMA_VERSION = "4.0.0";
 
-	@Activate
 	@Override
-	protected void activate() {
-		super.activate();
+	public String getSchemaVersion() {
+		return SCHEMA_VERSION;
+	}
 
+	@Activate
+	protected void activate() {
 		setDataLevel(DataLevel.PORTLET_INSTANCE);
+		setDataPortletPreferences(
```

Comment:
Hey @MarinhoFeliphe , sort the portletPreferences name alphabetically, please. Apply this for other places as well.

Thx